docs/source/courses/hpc-behaviour.md

```diff
@@ -11,14 +11,4 @@ Students should bring their own laptop with Python installed. They should also p
 an SWC username and password and be able to 
 [log into the SWC's HPC cluster](https://wiki.ucl.ac.uk/display/SSC/Logging+into+the+Cluster).
 If you require any assistance, please contact
-<a href="mailto:adam.tyson@ucl.ac.uk?subject=SWC/GCNU Software Skills">Adam Tyson</a> in advance of the course.
-
-## Instructors
-* [Adam Tyson](https://github.com/adamltyson)
-* [Niko Sirmpilatze ](https://github.com/niksirbi)
-* [Laura Porta](https://github.com/lauraporta)
-
-
-## Materials
```

Comment:
Slides and github repo should still be valid, no? I mean we'll update them and the links will stay the same.